docs/en/query_language/dicts/external_dicts_dict_layout.md

```diff
@@ -77,6 +78,18 @@ Configuration example:
 </layout>
 ```
 
+### sparse_hashed {#dicts-external_dicts_dict_layout-sparse_hashed}
```

Comment:
So we're adding the usage of sparse ht as a dictionary-wide setting, not attribute-wise? This makes sense, if we're going to change it to the one hash table per dictionary later. 
This makes me wonder -- do we really need `if (sparse)` everywhere? Instead of this, I'd add a hash table template parameter to the HashedDictionary -- no runtime conditions and much less changes in code.

Reply:
>Instead of this, I'd add a hash table template parameter to the HashedDictionary -- no runtime conditions and much less changes in code.

This was my first attempt, but then I decided that there will be too much templating

Reply:
`if (sparse)` is dirty but tolerable.